samples/app-sso/nodejs/README.md

```diff
@@ -0,0 +1,175 @@
+# Teams Tab, Bot, ME - search, action, linkunfurl SSO with Node JS
+
+__Tab SSO__
+This sample shows how to implement Azure AD single sign-on support for tabs. It will
+
+- Obtain an access token for the logged-in user using SSO
```

Comment:
Please add screenshot to setup connection string in App

Reply:
Entire setup is mentioned under 

## To try this sample

- Setup for Bot SSO
Refer to [Bot SSO Setup document](https://github.com/OfficeDev/Microsoft-Teams-Samples/blob/main/samples/bot-conversation-sso-quickstart/BotSSOSetup.md).

Please have a look and let me know if anything needs to be added

Reply:
Please mark review comment resolve once you commit changes